db/migrate/20170815231353_edu_benefits_data_migration.rb

```diff
@@ -0,0 +1,27 @@
+class EduBenefitsDataMigration < ActiveRecord::Migration
+  def change
+    # form_type is `1990 1995 1990e 5490 5495 1990n` all lowercase
+    insert_sql = <<-sql
+      INSERT INTO saved_claims
+        (encrypted_form, encrypted_form_iv, type, form_id, education_benefits_claim_id, guid)
+      SELECT
+        encrypted_form,
+        encrypted_form_iv,
+        concat('SavedClaim::EducationBenefits::VA', form_type),
+        concat('22-', upper(form_type)),
+        id,
+        #{ActiveRecord::Base::sanitize(SecureRandom.uuid)}
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/blob/edu-benefits-claims/app/models/saved_claim.rb#L31
not sure, i'm just setting the value the same way it's set in the model